File: doneBack/API/Models/Certificate.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace API.Models
{
  public class Certificate
  {
    public int CertificateId { get; set; }
    public string CourseId { get; set; }
    public string AccountId { get; set; }
    public DateTime GetDate { get; set; }
  }
}

File: doneWeb/src/app/model/certificate.ts
import { Course } from "./course";
export class Certificate{
    certificateId:number;
    courseId:string;
    accountId:string;
    getDate:string;
}
export class CertificateFullInfo{
    course:Course;
    userName:string;
    avatarPath:string;
    getDate:string;
}